test/sia.js

```diff
@@ -194,6 +196,36 @@ describe('sia.js wrapper library', () => {
 				expect(mockProcessObject.stdout.pipe.calledWith(fs.createWriteStream(Path.join('testdir', 'siad-output.log'))))
 			})
 		})
+		describe('call', () => {
+			it('should not leak file descriptors making heavy requests to unresponsive endpoints', (done) => {
+				const ndescriptors = () => readdir.readSync('/proc/'+process.pid+'/fd').length
```

Comment:
this test only works on linux, right? Ok for now, but if we start doing cross-platform testing it will need to be wrapped in a platform check.